README.md

```diff
@@ -3,3 +3,53 @@
 [Azure Static Web Apps](https://docs.microsoft.com/azure/static-web-apps/overview) allows you to easily build [Angular](https://angular.io/) apps in minutes. Use this repo with the [Angular quickstart](https://docs.microsoft.com/azure/static-web-apps/getting-started?tabs=angular) to build and customize a new static site.
 
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli).
+
+## Project setup
+
+```bash
+npm install
+```
+
+### Start the dev server
+
+```bash
+npm run swa:start
+```
+
+> Note: This command will use the local configuration file `swa-cli.config.json`.
+
+### Run unit tests

```

Comment:
Personal preference to not include test, e2e, and lint in the Readme. That's easily found by checking `npm run` and this kind of info tends to run out of date with the package.json over time. I'd only list install and swa commands. But not blocking.